hitl/run.sh

```diff
@@ -1,36 +1,38 @@
-#!/bin/bash
```

Comment:
Unrelated change. Either provide a rationale or remove.

Reply:
This is more portable since it doesn't require `bash` to be in `/bin`.

Reply:
Did you do that out of habit or does this really not work on your system?

Reply:
I this particular case out of habit, but I already encountered too many scripts that don't work with the outdated `/bin/bash` on macOS to get out of this habit. :smile: